esx_service/cli/vmdkops_admin.py

```diff
@@ -914,6 +913,13 @@ def generate_tenant_vm_ls_rows(vms):
 
 def tenant_vm_ls(args):
     """ Handle tenant vm ls command """
+
+    # Handling _DEFAULT tenant case separately to print info message
+    # instead of printing empty list
+    if (args.name == "_DEFAULT"):
```

Comment:
Can we use a constant instead of hard-coded string? I'd suggest to move all the constants below from auth.py to auth_data_const.py so that they can be shared by other modules:

DEFAULT_TENANT = '_DEFAULT'
DEFAULT_TENANT_UUID = '11111111-1111-1111-1111-111111111111'
DEFAULT_DS  = '_DEFAULT'
DEFAULT_DS_URL = DEFAULT_DS + "_URL"